pkg/query-service/utils/format.go

```diff
@@ -144,7 +144,7 @@ func ValidateAndCastValue(v interface{}, dataType v3.AttributeKeyDataType) (inte
 // ClickHouseFormattedValue formats the value to be used in clickhouse query
 func ClickHouseFormattedValue(v interface{}) string {
 	switch x := v.(type) {
-	case int:
+	case int, int32, int64:
```

Comment:
Add the same below for array `x[0].(type)` type cases. Also, It doesn't hurt to keep all integer types here.

Reply:
Added them